app/components/documents/document-card.hbs

```diff
@@ -259,4 +259,4 @@
   @confirmMessage={{t "delete"}}
   @confirmIcon="trash"
   @alert={{true}}
-/>
+/>
```

Comment:
Just a sidenote, as this wasn't specified in the original issue. When we delete a document container now without first deleting the signed piece, the signed document still lives in the db and on the system (it's now orphaned).

We have a follow-up issue for that (KAS-2794), but I'm wondering if we should disable the removal of pieces that have signed pieces for now. That would force the user to first delete the signed piece, and then the original.

Reply:
This is not the only place where you can remove document containers. (batch edit will also delete&cleanup / document-card actions..) 
I would argue we fix this separately